addons/arsenal/stringtable.xml

```diff
@@ -737,6 +743,7 @@
         </Key>
         <Key ID="STR_ACE_Arsenal_buttonClearContainerTooltip">
             <English>Empty the selected container</English>
+            <German></German>
```

Comment:
Empty entry

Reply:
"Der Selektierte Kontainer ist Leer"

Reply:
I don't think that's right. It's a command to empty the container.

Reply:
oh true, miss read it :D then it would be "Leert den Selektierten Kontainer"

Reply:
"Leert den ausgewählten Kontainer"